docs/development/API/Dshot.mdx

```diff
@@ -43,7 +50,9 @@ SSSSSSSSSSSTCCCC
 
 The interesting part is, that 1 and 0 in the DShot frame are distinguished by their high time. This means that every bit has a certain (constant) length, and the length of the high part of the bit dictates if a 1 or 0 is being received.
 
-![DShot high times](./dshot_bit.webp)
+<p align="center">
+  <img src={Bit} alt="DShot Bit" className="themeInvert" />
+</p>
```

Comment:
@VitroidFPV could we retain using markup instead of falling back using html with styling in content?

Reply:
This should work as a compromise I think:
```
<p align="center themeInvert">

    ![DShot 8bit](<URL>)

</p>
```
It's possible to put Markdown elements inside HTML elements, but there isn't a Markdown element to specifically center content AFAIK

Reply:
Unfortunately that didn't work for me. But maybe I'm just inept, so I could use some help 😅